CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - MANE badges on list of variant's Genes/Transcripts/Proteins table, this way also SVs will display MANE annotations
 - Export variant type and callers-related info fields when exporting variants from variants pages
 - Cases advanced search on the dashboard page
+- Possibility to use only signed off panels when building the PanelApp GREEN panel
```

Comment:
Not sure we will actually use that one, but seems very reasonable and prudent! Certification level stuff. 😁